### ISSUE TITLE
repeated print

### ISSUE THREAD

**Issue by undefinedcodezhong** [KillianLucas/open-interpreter#256]
**Describe the bug**
Sometimes, I encounter a situation where the same information needs to be printed multiple times like this: 
![image](https://github.com/KillianLucas/open-interpreter/assets/40236765/7ff030e0-ebbf-46e3-b5cb-845c5ef1c322)

**To Reproduce**

**Expected behavior**

**Screenshots**

**System (please complete the following information):**
 - OS: windows10
 - Python version 3.10
 - Open Interpreter version (`pip show open-interpreter`): 0.1.3 (I upgrade it by `pip install --upgrade --force-reinstall git+https://github.com/KillianLucas/open-interpreter.git
Collecting git+https://github.com/KillianLucas/open-interpreter.git`)

**Additional context**

**Comment by AshwinRamachandran2002** [comment]
were you using the Llama model?

**Comment by undefinedcodezhong** [comment]
> were you using the Llama model?

I use gpt-3.5-turbo model by '--fast' param. This is not a phenomenon that can be consistently reproduced. I am not sure if it is a code issue or a terminal environment issue.

**Comment by blackulaphoto** [comment]
yeah this happens all the time for me with gpt4 the bug is super expensive its actually a real problem it takes a job that should cost $1 and endss up costing $7 because it repeats everything 100 times

**Comment by undefinedcodezhong** [comment]
> yeah this happens all the time for me with gpt4 the bug is super expensive its actually a real problem it takes a job that should cost $1 and endss up costing $7 because it repeats everything 100 times

So expensive！
